erica/application/JobService/job_service_factory.py

```diff
@@ -78,6 +81,16 @@ def _send_est_injector():
         module
     ])
 
+def _send_grundsteuer():
+    module = ApplicationModule()
+    module.bind(Type[EricaRequestController], to_instance=GrundsteuerRequestController)
+    module.bind(Type[BasePayload], to_instance=GrundsteuerData)
```

Comment:
GrundsteuerData should be renamed GrundsteuerPayload, in accordance to the other uses cases. And as already mentioned, should be also moved to the application layer

Reply:
Done